readthedocs/redirects/models.py

```diff
@@ -78,13 +78,28 @@ class Meta(object):
         ordering = ('-update_dt',)
 
     def __str__(self):
+        redirect_text = '{type}: {from_url} -> {to_url}'
         if self.redirect_type == 'prefix':
-            return ugettext('Prefix Redirect:') + ' %s ->' % self.from_url
+            return redirect_text.format(
+                type=ugettext('Prefix Redirect'),
+                from_url=self.from_url,
+                to_url=''
```

Comment:
What about adding something like the docs say?

http://docs.readthedocs.io/en/latest/user-defined-redirects.html

    docs.example.com/dev/install.html -> docs.example.com/en/latest/install.html

So, maybe it could say

    Prefix Redirect: /dev -> {default_language}/{default_version